### DISCUSSION TITLE
Outposts and beacons float in editor test mode

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15085] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

They float, when ingame they are locked in place. the difference is slight, but it *does* affect physics and testing with spawned creatures. (a beacon floating upwards at 30km/hr is not really gonna be attacked by any creature, they're too slow)

having to do the lock, locky, or lockx commands every single time gets quite tedious, I think it should be enabled by default when saved as a beacon or outpost.

### Reproduction steps

1. make an outpost or beacon
2. test mode
3. go out the airlock
4. watch as it flies away from you at mach 5 :(

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately could not find a way to reproduce the issue. Outposts, beacon stations, outpost modules and wrecks seem to be static as expected. To be sure, you're not using any mods? Have you already saved the sub as an outpost/beacon (if not, the game will assume it's a normal sub)? 

https://github.com/user-attachments/assets/cd031a90-ec44-403b-8199-66a61421e565


https://github.com/user-attachments/assets/16e9c4c5-5883-49a0-8bc2-9ca638824f41

**Reply by girlyguppy** [reply]
nvm I imagined the bug
